config/locales/ja.yml

```diff
@@ -191,57 +191,57 @@ ja:
     title:
       reset_password: パスワードをリセットする
   medications:
-    singular: Medication
-    edit_medication: 'Edit Medication'
-    refill_reminder: 'Refill reminder email'
-    dosage: 'Daily Dosage:'
-    weekly_dosage: 'Weekly dosage:'
-    refill: 'Refill:'
-    quantity: 'Total Quantity:'
-    strength: 'Strength:'
+    singular: 処方薬
+    edit_medication: 処方薬を編集をする
+    refill_reminder: 補充リマインドメール
+    dosage: '一日投薬量：'
+    weekly_dosage: '一週間投薬量：'
+    refill: '補充：'
+    quantity: '量：'
+    strength: '強さ：'
     form:
-      strength: Strength
+      strength: 強さ
       strength_placeholder: 'e.g. 100'
       total_placeholder: 'e.g. 30'
-      dosage: dosage
+      dosage: 投薬量
       dosage_placeholder: 'e.g. 2'
-      refill: refill
-      comments_hint: 'Add any additional comments here, e.g. logging your symptoms'
-      add_to_google_cal: 'Google Calendar reminder'
-      google_cal_hint: 'We''ll add a reminder for this medication to your Google calendar.'
-      strength_hint: 'Medication strength, usually found beside the name'
-      total_hint: 'Total number of pills or amount of medication'
-      dosage_hint: 'Number of pills or servings daily'
-      weekly_dosage_hint: 'Days of the week to take this medication'
-      refill_hint: 'When to refill this prescription'
-      refill_reminder_hint: 'We''ll email you a reminder the week before your medication needs to be refilled.'
-      daily_reminder_hint: 'We''ll email you a daily reminder to take this medication'
-      total: Total
-    new: 'New Medication'
+      refill: 補充
+      comments_hint: 'ここにコメントしてください。例えば自分の症状を記録します。'
```

Comment:
```suggestion
      comments_hint: 'ここにコメントしてください。例えば自分の症状など
      など。'
```

Reply:
Should the suggestion have gone into a second line?